scripts/configure.js

```diff
@@ -508,7 +511,11 @@ const getConfig = (() => {
           },
         },
         windows: {
-          files: {},
+          files: {
+            ".gitignore": {
+               source: path.join(testAppPath, "example","windows", ".gitignore")
```

Comment:
Build is currently failing due to a formatting issue. Can you please run `yarn format:js`? That should fix it up.

![image](https://user-images.githubusercontent.com/4123478/124711055-e2e9d700-defd-11eb-98c7-aed836fb5b30.png)